yarn.lock

```diff
@@ -25553,7 +25553,7 @@ hexo-filter-github-emojis@arturi/hexo-filter-github-emojis:
   languageName: node
   linkType: hard
 
-"moment@npm:>= 2.9.0, moment@npm:^2.10.6, moment@npm:^2.22.2, moment@npm:^2.29.2":
+"moment@npm:>= 2.9.0, moment@npm:^2.10.6, moment@npm:^2.22.2, moment@npm:^2.29.4":
   version: 2.29.4
```

Comment:
Clearly we are already using `2.29.4`, I'm not sure I get what this PR is doing.

Reply:
Youre right. I guess I’m drunk too, like dependabot

Reply:
Don't drink and deploy